### PR TITLE
6 http requests

### PR DESCRIPTION
**Pull request by nikita-resh**
image
image

- add get&post requests to json placeholder

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+import { AppBar, Toolbar, Typography, Button } from "@mui/material";
 import ViewComfyIcon from "@mui/icons-material/ViewComfy";
 
 const iconStyles = { mr: 2 };
 
-const Header = () => {
+const Header = ({ cardList, setCardList }) => {
+  const handleAdd = async () => {
+    const randNum = Math.ceil(Math.random() * 100);
+    const { data: card } = await axios.get(`/posts/${randNum}`);
```
Comment (ArtsiomLysianok) [inline src/components/Header/Header.js:12]: Can you create an API layer for this purpose ?

File: src/axios/axios.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export default axios.create({
```
Comment (ArtsiomLysianok) [inline src/axios/axios.js:3]: If you move to another http client would you rename all that stuff ?

File: src/components/Header/Header.js (hunk 1 above)
Comment (nikita-resh) [reply src/components/Header/Header.js]: Done.

File: src/axios/axios.js (hunk 2 above)
Comment (nikita-resh) [reply src/axios/axios.js]: Fixed.

File: src/apiClient/apiClient.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import httpClient from "../httpClient/httpClient";
+
+export function apiClient() {
```
Comment (ArtsiomLysianok) [inline src/apiClient/apiClient.js:3]: It looks not like an API client... it looks like initApiClient function

Comment (nikita-resh) [reply src/apiClient/apiClient.js]: Fixed

**Review by ArtsiomLysianok — APPROVED** [review]

**Pull request merged**